pkg/component/controller/clusterConfig.go

```diff
@@ -121,7 +121,7 @@ func (r *ClusterConfigReconciler) Start(ctx context.Context) error {
 					r.log.Debug("config source closed channel")
 					return
 				}
-				err := multierr.Combine(cfg.Validate()...)
+				err := multierr.Combine(cfg.Validate(true)...)
```

Comment:
This seemed like a place where it should be `true`.

Reply:
I'd say this should be false here? Maybe only this controller has connectivity issues, but the others are fine...

Reply:
After letting this sink in a little bit more: The storage configuration is not a dynamic config setting anyways and should be always nilled out here. This is a technical debt, as k0s distinguishes "node config" and "cluster config" (aka dynamic config), but doesn't have distinct structs for both.